doc/nomenclature.rst

```diff
@@ -0,0 +1,12 @@
+============
+Nomenclature
+============
+
+Brackets
```

Comment:
would it make sense for you, @FriendsOfPHP/php-cs-fixer ?
i see that sometimes we mix braces/brackets/parentheses, maybe it's worth to unify?

Reply:
long road ahead to unify it fully, but it's good way to start 👍🏼

Reply:
@keradus 

I am for consistent naming, even if that requires renaming fixers and options!

Reply:
Do we want to rename (deprecate existing + introduce proxy) every fixer that uses old nomenclature within this PR, or maybe only discuss and prepare a list of fixers and options to rename, and create an issue or milestone for that?

Reply:
@Wirone 

I think it would be best to create an issue to keep track of the related and necessary changes, then open individual pull requests. 

I have already started as you can see in

- #7331 
- #7333 
- #7334 
- #7339